thoth/storages/graph/postgres.py

```diff
@@ -4163,6 +4164,9 @@ def sync_analysis_result(self, document: dict) -> None:
         environment_name = document["metadata"]["arguments"]["extract-image"]["image"]
         os_name = document["result"]["operating-system"]["name"]
         os_version = document["result"]["operating-system"]["version_id"]
+        cuda_version = document["result"]["cuda-version"]["/usr/local/cuda/version.txt"]
+        if cuda_version != document["result"]["cuda-version"]["nvcc_version"]:
+            raise CudaVersionDoesNotMatch
```

Comment:
Let's propagate this fact so that we see it in reporting tools:

```python
raise CudaVersionDoesNotMatch(f"Cuda version detect by nvcc {cuda_version!r} is different from the one found in /usr/local/cuda/version.txt {document['result']['cuda-version']['nvcc_version']!r}")
```

Reply:
Thanks, done!